go.mod

```diff
@@ -0,0 +1,71 @@
+module github.com/armory-io/terraform-provider-spinnaker
+
+go 1.12
+
+require (
+	cloud.google.com/go v0.39.0
+	github.com/agext/levenshtein v1.2.2
+	github.com/apparentlymart/go-cidr v1.0.0
+	github.com/apparentlymart/go-textseg v1.0.0
+	github.com/armon/go-radix v1.0.0
+	github.com/aws/aws-sdk-go v1.19.42
+	github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d
+	github.com/bgentry/speakeasy v0.1.0
+	github.com/blang/semver v0.0.0-20190414102917-ba2c2ddd8906
+	github.com/estebangarcia/spin v0.0.0
```

Comment:
This should use spinnaker/spin, not a fork. When using v0.0.0, you should also format it as `v0.0.0-${DATE}-${COMMIT_12}`. For the LDAP changes that're upstream, this would be `v0.0.0-20190530150642-535d2dc1b985` for the current master.